src/test/java/catalinazoldiSeleniumBasics2/teorie/SBpart3/pageObject/EmagHomePage.java

```diff
@@ -0,0 +1,29 @@
+package catalinazoldiSeleniumBasics2.teorie.SBpart3.pageObject;
```

Comment:
Please move `SBpart3`package in `src/test/java/catalinazoldi/teorie` package. After you move the `SBpart3`, please remove catalinazoldiSeleniumBasics2 package from `src/test/java`. Thanks!
![image](https://user-images.githubusercontent.com/18215854/222141441-38fe37fa-97bb-4181-a7e0-1a9a958e9939.png)